PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1387,7 +1387,7 @@ Translation note:
 				<Item id="1720" name="&amp;. matches newline"/>
 			</FindInFinder>
 			<DoSaveOrNot title="Save">
-				<Item id="1761" name="Save file &quot;$STR_REPLACE$&quot; ?"/>
+				<Item id="1761" name="Save file &quot;$STR_REPLACE$&quot; ?"/><!-- Don't translate $STR_REPLACE$ -->
```

Comment:
you shouldn't modify this file.

Reply:
Ok, I will revert. Thought this change is needed.
Btw, I need to rewrite those changes on latin and PR will be ready for review.

Reply:
The modified xml are not validated XML files.
Please fix them.